### PR TITLE
Added the 'Readex Pro' font

### PR DESCRIPTION
**Pull request by mortcodesweb**
Affected apps:
- Camera
- Dialer
- Files
- First Time Use (FTU)
- Home Screen
- Keyboard
- Messages
- Settings
- System
- Terminal (Only Open Terminal Screen)
- UI Test

Unaffected:
- Shoelace

Bonus Changes:
- Capitalized the C in the Camera app name
- Capitalized the S in the Settings app name
- FTU app name is now capitalized.

### CODE REVIEW

File: apps/camera/style/index.css [hunk 1]
```diff
@@ -4,7 +4,7 @@ body {
   height: 100%;
   margin: 0;
   padding: 0;
-  font-family: "Open Sans", arial, "sans serif";
+  font-family: 'Readex Pro', sans-serif;
```
Comment (fabricedesre) [inline apps/camera/style/index.css:7]: please use double quotes for strings litteral

File: apps/system/components/input_method.css [hunk 2]
```diff
@@ -12,7 +12,7 @@ input-method {
 }
 
 input-method:not(.no-animation) {
-  transition: transform 0.15s;
+  transition: transform 0.3s cubic-bezier(0.2, 0.9, 0.1, 1);
```
Comment (fabricedesre) [inline apps/system/components/input_method.css:15]: unrelated change, please revert in this PR

File: apps/system/js/audio_volume/linux.js [hunk 3]
```diff
@@ -47,7 +47,7 @@ async function setupAudio() {
   }
 
   // Initialize the HW to the current value.
-  await setAudioVolume(currentVolume);
+  // await setAudioVolume(currentVolume);
```
Comment (fabricedesre) [inline apps/system/js/audio_volume/linux.js:50]: unrelated change, please revert in this PR

**Review by fabricedesre — CHANGES_REQUESTED** [review]
Please address comments. Is that properly applying to the text in shoelace components?

File: apps/camera/style/index.css (hunk 1 above)
Comment (mortcodesweb) [reply apps/camera/style/index.css]: alright

File: apps/system/components/input_method.css (hunk 2 above)
Comment (mortcodesweb) [reply apps/system/components/input_method.css]: oh i forgot about that

File: apps/system/js/audio_volume/linux.js (hunk 3 above)
Comment (mortcodesweb) [reply apps/system/js/audio_volume/linux.js]: alright

**Comment by mortcodesweb** [comment]
It dosen't apply for shoelace because shoelace has its own font familes set.

File: apps/shared/style/fonts.css [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  src: url(/fonts/Readex_Pro/ReadexPro-VariableFont_wght.ttf);
+}
+
+* {
+  --sl-font-sans: "Readex Pro", sans-serif;
+}
+
+html, body {
+  font-family: var(--sl-font-sans);
+}
```
Comment (fabricedesre) [inline apps/shared/style/fonts.css:12]: Since you have a rule here for html and body, all the ones adding the same font for the same tags in style.css are redundant, right? If so they should be removed instead of updated.

Comment (mortcodesweb) [reply apps/shared/style/fonts.css:14]: yes
